pkg/cli/cliflags/flags.go

```diff
@@ -255,8 +255,11 @@ production usage.`,
 	ServerInsecure = FlagInfo{
 		Name: "insecure",
 		Description: `
-Start an insecure node, using unencrypted (non-TLS) connections. This is strongly discouraged for
-production usage.`,
+Start an insecure node, using unencrypted (non-TLS) connections,
+listening on all IP addresses and disabling password authentication
+for all database users. This is strongly discouraged for production
+usage and should never be used on a public network without combining
+it with --host and --http-host.`,
```

Comment:
To be fair, the admin UI doesn't contain much sensitive info. It might be better to say that it should never be used on a public network with data that you care about at all.